src/aggregator/integration/custom_aggregations_test.go

```diff
@@ -94,6 +94,14 @@ func testCustomAggregations(t *testing.T, metadataFns [4]metadataFn) {
 	require.NoError(t, setPlacement(placementKey, placementStore, placement))
 	serverOpts = serverOpts.SetPlacement(placement)
 
+	// Topic setup
+	m3msgTopicName := defaultTopicName
+	topicService, err := initializeTopic(m3msgTopicName, serverOpts.KVStore(), placement)
+	require.NoError(t, err)
+	serverOpts = serverOpts.
+		SetTopicService(topicService).
+		SetTopicName(m3msgTopicName)
```

Comment:
These 7 lines are copied in 7 places, could we make this a function?

Reply:
https://github.com/m3db/m3/pull/3681/commits/a8a41310c77e1282ccda630c6885253489f162a6